tools/diagnostic-information

```diff
@@ -97,6 +97,30 @@ image_version(){
     fi
 }
 
+licensing_information(){
+    verbose "licensing information..."
+    title 1 "licensing information"
+
+    # Coreutils timeout
+    TIMEOUT=""
+    if which timeout 2>/dev/null >/dev/null ; then
+        TIMEOUT="timeout 5s "
+    fi
+
+    # Notes:
+    # * bmsg request does not return 0 for licensing INFO, since the MB does
+    #   not return "OK", so trap it using " || :" to not have the script aborting
+    # * also delete everything before the first actual information
+    $TIMEOUT bmsg -t 5 request etn-licensing GET REQUEST 1234 INFO 2>/dev/null | sed -n '/overall-state/,$p' 1>/tmp/licensing_info || :
+
+    if [ -f /tmp/licensing_info ] ; then
+        quote < /tmp/licensing_info
+        rm -f /tmp/licensing_info
+    else
+        echo -e "Error: Missing licensing information\n"
```

Comment:
Why `echo -e "something\n"` instead of `echo "something"`? :)
And I found over the years that `echo -e` is not portable, some shells have it, some don't, some call a built-in, others a `/bin/echo`... if really needed, better call `printf` that is intended for such extended formatting.

Reply:
for the echo, I've simply aligned with what was already done (such as in image_version()). So for now, leaving this as is, log a ticket to address portability Vs formatting, and we'll see later.